CryptoWorld.News/CryptoWorld.News.Server/Controllers/UserProfileController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using CryptoWorld.News.Core.Services;
+using CryptoWorld.News.Core.ViewModels;
+using CryptoWorld.News.Data;
+using CryptoWorld.News.Data.Models;
+using CryptоWorld.News.Core.Interfaces;
+using CryptоWorld.News.Core.ViewModels;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace CryptoWorld.Application.Server.Controllers
+{
+    [Authorize]
+    public class UserProfileController : BaseApiController
+    {
+        private readonly IUserProfileService profileService;
+        private readonly ApplicationDbContext context;
+        private readonly UserManager<ApplicationUser> userManager;
+        public UserProfileController(IUserProfileService _profileService, ApplicationDbContext _context, UserManager<ApplicationUser> _userManager)
```

Comment:
private field should be with _ like ---->   private readonly UserManager<ApplicationUser> _userManager; not in the constructer. Please fix whole PR

Reply:
I mean every private fields